examples/run_tf_glue.py

```diff
@@ -1,105 +1,240 @@
+# coding=utf-8
+""" Fine-tuning the library models for named entity recognition."""
```

Comment:
```suggestion
""" Fine-tuning the library models for sequence classification."""
```

Reply:
Ahah the usual miss of copy/paste 😆 Will be updated for the next commit ^^